libsoftwarecontainer/src/gateway/gateway.cpp

```diff
@@ -21,11 +21,30 @@
 
 namespace softwarecontainer {
 
+
+Gateway::Gateway(const std::string &id,
+                 std::shared_ptr<ContainerAbstractInterface> container,
+                 bool quickLaunchEnabled) :
+    m_id(id),
+    m_container(container),
+    m_quickLaunchEnabled(quickLaunchEnabled),
```

Comment:
I thought about this. The concept of quick-launch is a bit abstract for the Gateway interface, I agree with that. But the interface also says that one can configure and activate, and this change is about re-configuring and re-activating, so calling it "reconfigurable" will not match up well to the interface. Do you have a suggestion for a name that covers this?